lib/config.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Bedrock Protractor Module Configuration
```

Comment:
personally I don't like these descriptor lines for this very reason.  Remove or fix 'Protractor'